2.05-custom-gx/map_user.hsp

```diff
@@ -643,13 +643,19 @@
 		gosub *prompt_word
 		if ( inputlog == "" ) {
 			txt lang("名前をつけるのはやめた。", "You changed your mind.")
-			adata(ADATA_FESTIVAL, gdata(GDATA_AREA)) = 0
+			//adata(ADATA_FESTIVAL, gdata(GDATA_AREA)) = 0
```

Comment:
What about the case when the player wants to remove the custom name of the current map? I think that could be the purpose of this code.

Reply:
In the unmodified game, entering nothing in the prompt doesn't reset the name to default. I think that might be nice functionality to have, but if that's supposed to be the purpose of this code, it doesn't work. In any event, the player can just enter "Shop", "Crop", etc. and achieve the same thing so I didn't think it was important to add.